### ISSUE TITLE
readme中未找到启动训练的shell脚本和json文件

### ISSUE THREAD

**Issue by Venomfly** [mindspore-lab/mindcv#335]
涉及的URL：https://github.com/mindspore-lab/mindcv/tree/main/configs/mobilenetv3
readme中未找到生成run_distribute_train_ascend.sh 和hccl_8p_01234567_123.60.231.9.json文件的说明，请补充。
![image](https://user-images.githubusercontent.com/99794814/209526939-65a81673-36c0-4c1f-a68f-b8139eeaf6c2.png)

**Comment by vigo999** [comment]
please use python train.py as example. no json file

**Comment by geniuspatrick** [comment]
duplicate issue of #329  , we need to review the "distributed training command" part of README file for all models.

**Comment by geniuspatrick** [comment]
A recommended solution is to change all distributed training commands to `mpirun` mode. As follows:

```shell
mpirun -n 8 python train.py -c /path/to/config.yaml --data_dir=/path/to/data
```

All subsequent new models' PR should follow this rule.

**Comment by geniuspatrick** [comment]
fixed in #362 #367 #368  #369 #370
